src/db.js

```diff
@@ -34,7 +37,8 @@
                 args = args || [];
                 state = state || [ context , args ];
 
-                for ( var i = 0 , il = list.length ; i < il ; i++ ) {
+                var i, il;
+                for ( i = 0 , il = list.length ; i < il ; i++ ) {
```

Comment:
I'd prefer this not be changed, my coding preference is to declare the variables in the `for` as part of themselves